packages/camera/camera_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.1
+
+* Added CameraOptions used to constrain the camera audio and video.
```

Comment:
We aren't still publishing this, so no need to bump any versions. right?

Reply:
True, I followed [these guidelines](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates) but since we are not publishing it yet, we may skip bumping the version and updating the changelog as you say.

Reply:
Yes, we can just keep adding stuff to 0.1.0 until we actually publish the first version, and then it's when we need to update the version and changelog to make it available to people.

There's the NEXT version too supported in the changelog that allows us to push fixes that don't need to be published right away (mostly they are tooling/repo maintenance changes with no real changes in user features)